release/release_after_kokoro.sh

```diff
@@ -102,7 +102,7 @@ ANALYTICS_CONSTANT=$( javap -classpath \
 echo -e "$LOGIN_CONSTANTS\n$ANALYTICS_CONSTANT"
 
 # Verify if the constants have been injected.
-if [[ "${LOGIN_CONSTANTS}${ANALYTICS_CONSTANT}" =~ "@" ]]; then
```

Comment:
FTR, I'm happy with bash'isms for this script — that's nice and compact. Just be sure to change the first line to `#!/bin/bash` though.